packages/devtools/lib/src/timeline/timeline.dart

```diff
@@ -127,6 +131,11 @@ class TimelineScreen extends Screen {
             pauseButton,
             resumeButton,
           ]),
+        PButton.icon('Clear', _clear)
```

Comment:
perhaps have the clear button with aligned w/ the side side of the page?

Reply:
I'm guessing you meant "right side"? If so, I think it makes more sense to be grouped with pause/resume than with export (and service extension buttons in the future).